src/test/java/org/mongojack/TestDBUpdateSerialization.java

```diff
@@ -182,4 +206,71 @@ public void serialize(String value, JsonGenerator jgen,
             }
         }
     }
+    
+    public static class NestedRepeatedAttributeName {
+    	public static class Inner {
+    		public Date timestamp;
+
+			@Override
+			public int hashCode() {
```

Comment:
are the hashCode and equals necessary? you don't have them in the example in https://github.com/devbliss/mongojack/issues/98